pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -63,16 +61,6 @@ func (h *Helper) GetDockerAuthSearchPaths(authType string) []string {
 	return searchPaths
 }
 
-// SetSystemContextFilePath properly seeds the container/image SystemContext
```

Comment:
So yeah this got introduced for getting the image registry docker secret (which was in the legacy format) to work the containers/image registries.conf stuff when we added disconnected / mirror support in 4.2

Reply:
I understand. As I am saving the "merged pull credentials" file using the new format I did not see why to keep this function around(it is not used anywhere inside this project). Is this used somewhere else?

Reply:
It is called from this project:  https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L61

Reply:
Yeah, that line was removed with this PR.

Reply:
cool, which I think makes sense given the intent of this PR